### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by alexander-fenster**
Fixes failing test for node4 and node6. 

- [x] `npm test` succeeds
- [x] Pull request has been squashed into 1 commit
- [x] I did NOT manually make changes to anything in `apis/`
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate JSDoc comments were updated in source code (if applicable)
- [x] Appropriate changes to readme/docs are included in PR

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
That's weird 😂

**Comment by JustinBeckwith** [comment]
Line 198 of test.auth.ts, we create nock for the old oauth token endpoint.  Please just replace that code with the new code you've added :)

**Comment by alexander-fenster** [comment]
done, PTAL

File: test/test.auth.ts [hunk 1]
```diff
@@ -210,6 +206,12 @@ describe('OAuth2 client', () => {
     };
 
     nock(Utils.baseUrl).get('/drive/v2/files/wat').reply(200);
+    let scope = nock(Utils.baseUrl).post('/oauth2/v4/token').reply(200, {
```
Comment (JustinBeckwith) [inline test/test.auth.ts]: Sorry for missing this earlier.  You can just use `.times(s)` and a `const` here to ensure it runs twice :)

Comment (alexander-fenster) [reply test/test.auth.ts]: After some looking at the code, I guess the point is that it should **not** run at all. That's what happens in node8 and node9, and what does not happen in node4 and node6.

**Comment by JustinBeckwith** [comment]
Going to go ahead and close this one out since we're back up.

**Pull request closed without merging**